react/features/app/middleware.js

```diff
@@ -29,6 +31,32 @@ MiddlewareRegistry.register(store => next => action => {
     return next(action);
 });
 
+/**
+ * Notifies the feature app that the action {@link CONFERENCE_RESTARTED} is
+ * being dispatched within a specific redux {@code store}.
+ *
+ * @param {Store} store - The redux store in which the specified {@code action}
+ * is being dispatched.
+ * @param {Dispatch} next - The redux {@code dispatch} function to dispatch the
+ * specified {@code action} to the specified {@code store}.
+ * @param {Action} action - The redux action {@code CONFERENCE_RESTARTED}
+ * which is being dispatched in the specified {@code store}.
+ * @private
+ * @returns {Object} The new state that is the result of the reduction of the
+ * specified {@code action}.
+ */
+function _conferenceRestarted({ dispatch, getState }, next, action) {
+    const { enableForcedReload } = getState()['features/base/config'];
+
+    if (enableForcedReload) {
+        dispatch(reloadNow());
+
+        return;
```

Comment:
> Left some comments/questions.
> 
> In addition, don't we want to handle this situation on mobile too?

Thank you comments and suggestions. This should work on mobile too. Since mobile doesn't support prejoin page, I made sure we are not calling prejoin page related actions on mobile.

Reply:
On the initial version of the PR the action was not dispatched from mobile. Anyway verified the new version should work on mobile!